src/engraving/accessibility/accessibleitem.cpp

```diff
@@ -206,7 +208,43 @@ QString AccessibleItem::accessibleDescription() const
         return QString();
     }
 
-    return readable(m_element->accessibleExtraInfo());
+    Selection selection = m_element->score()->selection();
+
+    if (!selection.isRange() || !selection.startSegment() || !selection.endSegment()) {
```

Comment:
Since you use `startSegment()` and `endSegment()` later on, it's best to store them in a variable rather than calling them repeatedly.